lib/ansible/modules/network/aci/aci_access_port_to_interface_policy_leaf_profile.py

```diff
@@ -39,29 +39,53 @@
     - The description to assign to the C(access_port_selector)
   leaf_port_blk:
     description:
+    - B(Deprecated)
```

Comment:
I don't like this way of deprecating parameters. In fact the parameters can be deprecated by adding something to the argument_spec. And what we usually do is not mention the parameters in the documentation so people won't start using it (the people still using it get a deprecation warning).

Reply:
Ok. So it would be better to remove them completely from the docstring. But can you give me some more hints how exactly you think it would be done then (especially the deprecation warning)?